pandas/core/indexes/interval.py

```diff
@@ -191,10 +192,12 @@ class IntervalIndex(ExtensionIndex):
     _typ = "intervalindex"
 
     # annotate properties pinned via inherit_names
-    closed: str
+    closed: Literal["left", "right", "both", "neither"]
```

Comment:
If you like to, you could define this literal in _typing, is/will be used in many places.

Reply:
good idea.  done in next commit